added JS_HW_6 and JS_HW_7

AkhmerovYaroslav opened the pull request:
Добавлены домашки 6 и 7

AkhmerovYaroslav committed:
added JS_HW_6 and JS_HW_7

VladimirLQA commented on HW_6/JS_HW_6_Task2.js:
```diff
@@ -0,0 +1,41 @@
+
+
+
+for (let i = 0; i < myPizzas.length; i++) {
```
Попробуй немного переписать
- списки изначально приводим в нижний регистр
- затем отдельньім циклом бежим и проверяем, есть ли пицца у конкурента, можно воспользоваться `includes` методом и добавляем в наш массив результатов, если удовлетворяет условие

VladimirLQA commented on HW_6/JS_HW_6_Task2.js:
```diff
@@ -0,0 +1,41 @@
+const myPizzas = ['Пепперони', 'Капричоза', 'Diablo', '4 Сыра', 'Гавайская'];
+// const myPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+
+const PizzasOnlyIHave = [];
```
```suggestion
const pizzasOnlyIHave = [];
```

VladimirLQA commented on HW_6/JS_HW_6_Task3.js:
```diff
@@ -0,0 +1,15 @@
+
+const arrayOfNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 2, 4, 6, 8, 10, 1, 3, 5, 7, 9];
+
+const SetOfNumbers = new Set(arrayOfNumbers);
```
```suggestion
const setOfNumbers = new Set(arrayOfNumbers);
```

VladimirLQA commented on HW_7/JS_HW_7_Task1.js:
```diff
@@ -0,0 +1,81 @@
+*/
+
+function getDevidedBy_(sentence) {
+    const words = sentence.split(' ');
+    const devideBy_sentence = words.map((word, i) => {
+      if (i === 0) {
+        return word.charAt(0).toLowerCase() + word.slice(1);
+    } else {
+        return word.charAt(0).toUpperCase() + word.slice(1);
+    }
+  }).join('_');
```
Изначально приводим в нижний регистр для подобньіх случаев `bROtHeR`
```suggestion
    const words = sentence.toLowerCase().split(' ');
    return words.map((word, i) => {
      if (i !== 0) {
        return word.charAt(0).toUpperCase() + word.slice(1);
    }
        return word;
  }).join('_');
```

AkhmerovYaroslav replied:
Принял. Задание, не верно понял, решил, что речь идет исключительно про первые буквы слов.

VladimirLQA commented on HW_7/JS_HW_7_Task2.js:
```diff
@@ -0,0 +1,61 @@
+*/
+
+function checkForPalindrome(word) {
+  const wordToLowerCase = word.toLowerCase();
+  const wordSplitted = wordToLowerCase.split('');
+  const wordBackwards = wordSplitted.reverse().join('');
+  return wordToLowerCase === wordBackwards;
```
```suggestion
  const wordToLowerCase = word.toLowerCase();
  return wordToLowerCase === wordToLowerCase.split('').reverse().join('');
```

AkhmerovYaroslav committed:
Update HW_6/JS_HW_6_Task2.js

const PizzasOnlyIHave changed to pizzasOnlyIHave

Co-authored-by: Volodymyr Lazurenko <103571439+VladimirLQA@users.noreply.github.com>

AkhmerovYaroslav committed:
Update HW_6/JS_HW_6_Task3.js

const SetOfNumbers changet to setOfNumbers

Co-authored-by: Volodymyr Lazurenko <103571439+VladimirLQA@users.noreply.github.com>

AkhmerovYaroslav committed:
Update HW_7/JS_HW_7_Task2.js

wordToLowerCase declaration changed to wordToLowerCase === wordToLowerCase.split('').reverse().join('');

Co-authored-by: Volodymyr Lazurenko <103571439+VladimirLQA@users.noreply.github.com>

AkhmerovYaroslav committed:
Update HW_7/JS_HW_7_Task1.js

changes to function getDevidedBy_

Co-authored-by: Volodymyr Lazurenko <103571439+VladimirLQA@users.noreply.github.com>
